ngx_http_sticky_misc.c

```diff
@@ -20,8 +20,8 @@
 #define MD5_CBLOCK  64
```

Comment:
The #ifndef SHA_DIGEST_LENGTH conditional block is unnecessary because SHA_DIGEST_LENGTH is already defined.
The original code defines SHA_CBLOCK and SHA_DIGEST_LENGTH twice, which is redundant.

```c
/* - fix for 1.11.2 removes include <openssl/md5.h> in ngx_md5.h */
#define MD5_CBLOCK  64
#define MD5_LBLOCK  (MD5_CBLOCK / 4)
#define MD5_DIGEST_LENGTH 16

#define SHA_CBLOCK 64
#define SHA_DIGEST_LENGTH 20
```